alpine-nix-rails-nginx/etc/nginx/nginx.conf

```diff
@@ -79,20 +84,24 @@ http {
 
   # Catch-all domain
   server {
-    listen      80 default_server;
-    listen [::]:80 default_server;
+    listen      8080 default_server;
+    listen [::]:8080 default_server;
+
+    access_log /var/log/nginx/access.log;
+    error_log  /var/log/nginx/error.log;
```

Comment:
I should double check that this `error_log` is necessary. Maybe the config will just "inherit" from the outer-most declaration in the main context.